src/Utopia/Messaging/Adapters/SMS/Telnyx.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Utopia\Messaging\Adapters\SMS;
+
+use Utopia\Messaging\Adapters\SMS as SMSAdapter;
+use Utopia\Messaging\Messages\SMS;
+
+class Telnyx extends SMSAdapter
+{
+    /**
+     * @param  string  $accountSid Twilio Account SID
+     * @param  string  $authToken Twilio Auth Token
+     */
```

Comment:
Please update these to match the constructor params

Reply:
> You can run `composer format` to fix the linting error

Thanks!

> Please update these to match the constructor params
 
Whoops, not sure how I missed those.